lib/types.ts

```diff
@@ -15,6 +16,7 @@ export interface ContributorStats {
   score?: number
   approvalsGiven: number
   rejectionsGiven: number
+  distinctPRsReviewed?: number // Number of unique PRs reviewed by this contributor
```

Comment:
Google convention naming, the first letter of each word or acronym is capitalized (except the first) but no others

```suggestion
  distinctPrsReviewed?: number // Number of unique PRs reviewed by this contributor
```